src/nvim/eval.lua

```diff
@@ -1296,7 +1296,7 @@ M.funcs = {
 
     ]=],
     name = 'col',
-    params = { { 'expr', 'string|integer[]' }, { 'winid', 'integer' } },
+    params = { { 'expr', 'string|table' }, { 'winid', 'integer' } },
```

Comment:
It's a list so `any[]` gives slightly more info, right? Same for the other cases.

```suggestion
    params = { { 'expr', 'string|any[]' }, { 'winid', 'integer' } },
```